lib/cli/scripts/scan-env.ts

```diff
@@ -18,6 +18,11 @@ let loginAttempts: number = 0;
 
 export default async function main(_args: string[]) {
 
+    // Pre-commit detects console.log as an attempt to call the function, when we just want to assign an empty function to it
```

Comment:
https://palantir.github.io/tslint/usage/rule-flags/ you just need tslint disable next line

Reply:
I did not know about this. Thanks, changing right away